src/Main.elm

```diff
@@ -148,7 +149,7 @@ type alias FormField =
 
 
 type InputField
-    = ShortText String (Maybe Int)
+    = ShortText String (List ( String, String ))
     | LongText (Maybe Int)
     | Dropdown (List Choice)
     | ChooseOne (List Choice)
```

Comment:
why only add `List (String, String)` for `ShortText` variant only?

Reply:
initially the PR was to add multiple for `input[type="email"]` 😅 

so i change `LongText` as well?

Reply:
Might be good to take a step back and reassess the new state

Reply:
I see now that [`shortTextTypeList`](https://github.com/choonkeat/tiny-form-fields/blob/52082559701c1b015e9eacae477c907bdda37787/index.html#L56) config is only wired to add [`ShortText` variations in the form builder dropdown](https://github.com/choonkeat/tiny-form-fields/blob/48f0beec1ad060e78dd8d769b8db0bb0ab05636a/src/Main.elm#L953-L956)

![Screenshot 2024-09-13 at 8 38 41 AM](https://github.com/user-attachments/assets/14e6d637-5210-40fe-9da4-b528098d397c)

Then I think only adding `List (String, String)` to `ShortText` variant is fine 👌